ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
+import { BaseAvatar } from '../base-avatar';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+export const diameters = {
+  xs: '16',
+  sm: '24',
+  md: '32',
+  lg: '40',
+  xl: '48',
+};
+
+export const avatarAccountTypes = ['Jazzicon', 'BlockieIdenticon'];
+
+const getStyles = (diameter) => ({
+  height: diameter,
+  width: diameter,
+  borderRadius: diameter / 2,
+});
+
+export const AvatarAccount = ({
+  size,
+  address,
+  className,
+  avatarAccountType,
```

Comment:
Could this be simply `type`?

Reply:
I have followed the mobile convention here but I have replaced it with `type` now